index.html

```diff
@@ -82,24 +84,23 @@ <h2 class="visually-hidden">Услуги</h2>
         <li class="services-item">
           <h3 class="services-title">Жильё</h3>
           <p class="services-description">Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
-          <img class="services-image1" src="#" alt="Иконка домика">
+          <img class="services-image1" src="#" alt="#">
```

Comment:
Лучше так, скроём от Скрин Ридера
```suggestion
          <img class="services-image1" src="#" alt="" aria-hidden="true">
```